### PR TITLE
getting ncp with npm install

### PR DESCRIPTION
**Pull request by asafonov**
Setting ncp as a project dependency and use it in a build script. It's not a bad idea, isn't it?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,5 +20,8 @@
     "start": "node ./tools/start.js --debug=5859 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck",
     "build": "node ./tools/buildPackage.js",
     "lint": "standard"
+  },
+  "dependencies": {
```
Comment (diracdeltas) [inline package.json]: i think this should go in devDependencies instead of dependencies

**Review by diracdeltas — CHANGES_REQUESTED** [review]

**Comment by asafonov** [comment]
@diracdeltas I completely agree with you. Updated

**Comment by diracdeltas** [comment]
On second thought, it doesn't seem worth the effort to require `ncp` as a devDependency just for a single command where `cp -r` would work fine. Maybe just change it to `cp`?

**Comment by asafonov** [comment]
@diracdeltas Yes, the less dependencies the better. I changed build script, but I can't test on Windows

**Comment by diracdeltas** [comment]
thanks!

**Review by diracdeltas — APPROVED** [review]

**Pull request merged**